tests/jballerina-unit-test/src/test/resources/testng.xml

```diff
@@ -806,8 +806,8 @@
             </class>
             <class name="org.ballerinalang.test.expressions.literals.IdentifierLiteralPackageTest">
```

Comment:
https://github.com/ballerina-platform/ballerina-lang/pull/26807/files#r518952028 this comment applies here too.

Reply:
Removed